internal/dto/accountDto.go

```diff
@@ -0,0 +1,12 @@
+package dto
+
+type AccountDto struct {
```

Comment:
The preferred way in Go would be not to name structs with Dto suffix or similar.
The import path determines what you really want to do with it, and since the package accessor is always used to specify the type you are referring to, `dto.AccountDto` does not add more readability instead of `dto.Account`.